exp/addons/controllers/clusterresourceset_helpers.go

```diff
@@ -54,7 +54,7 @@ func isJSONList(data []byte) (bool, error) {
 	return bytes.HasPrefix(trim, jsonListPrefix), nil
 }
 
-func apply(ctx context.Context, c client.Client, data []byte) error {
+func apply(ctx context.Context, s addonsv1.ClusterResourceSetStrategy, c client.Client, data []byte) error {
```

Comment:
`s` is not very descriptive.

```suggestion
func apply(ctx context.Context, strategy addonsv1.ClusterResourceSetStrategy, c client.Client, data []byte) error {
```

Reply:
I am trying to follow style here, c for client isnt either.

Reply:
`c` is a common in the k8s codebase used as client.
lets please follow similar to the other param `data`.